packages/parser/test/Render_test.t

```diff
@@ -0,0 +1,4 @@
+  $ alias run='./Render_test.exe'
+
+  $ cat ./test.css | run
```

Comment:
why the alias?

Reply:
haha i copied the test from andrey's repo and forgot to change hehe